### PR TITLE
FTT-13678: Improve development logging

### PR DESCRIPTION
**Pull request by MohammedFoysal**
FTT-13678: Improve development logging

- Coloured levels
- json pretty print
- Add file writer
- Add optional config vars to enable/disable pretty print and metadata

Debug logs with pretty print
image

Debug logs with metadata disabled
image

File logs
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dvsa/azure-logger",
-  "version": "5.0.0",
+  "version": "5.0.1",
```
Comment (pjblakey) [inline package.json:3]: Feels like quite a big change, so minor or even major version bump?

File: src/helpers/getDate.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const getDateFilename = (): string => {
```
Comment (m-szyszka) [inline src/helpers/getDate.ts:1]: please add unit test

File: package.json (hunk 1 above)
Comment (MohammedFoysal) [reply package.json]: opted for a minor bump. there shouldn't be any breaking changes so major shouldn't be required

File: src/helpers/getDate.ts (hunk 2 above)
Comment (MohammedFoysal) [reply src/helpers/getDate.ts:1]: unit test added

**Review by pjblakey — APPROVED** [review]

**Review by m-szyszka — APPROVED** [review]

**Comment by TomBillingtonUK** [comment]
Can we have some updates in the readme, the new config values, how it works ...

**Pull request merged**